iroh-qrpc-types/README.md

```diff
@@ -0,0 +1,24 @@
+# iroh-rpc-types
+
+This crate defines types for use by the
+[iroh-rpc-client](https://github.com/n0-computer/iroh/tree/main/iroh-rpc-client),
+which is used for [iroh](https://github.com/n0-computer/iroh) services to
+communicate internally via RPC. The protocol used is [gRPC](https://grpc.io/).
+This crate defines the gRPC types in the form of [Protocol
+Buffers](https://developers.google.com/protocol-buffers). It uses the [Tonic
+framework](https://github.com/hyperium/tonic) to expose the gRPC types to Rust.
```

Comment:
blurb probably needs updating

Reply:
Right. I updated the one for client, but not this one.